R/shuffle_ard.R

```diff
@@ -223,8 +224,119 @@ shuffle_ard <- function(x, trim = TRUE) {
   }
 }
 
+#' Rename ARD Columns
```

Comment:
Can we put this function in its own R file? (Same applies to the unit testing file)

Reply:
done!